Userland/Libraries/LibJS/Runtime/GlobalObject.cpp

```diff
@@ -304,6 +304,7 @@ void GlobalObject::initialize_global_object()
     m_array_prototype_values_function = &m_array_prototype->get_without_side_effects(vm.names.values).as_function();
     m_date_constructor_now_function = &m_date_constructor->get_without_side_effects(vm.names.now).as_function();
     m_eval_function = &get_without_side_effects(vm.names.eval).as_function();
+    m_json_parse_function = &get_without_side_effects(vm.names.JSON)->get_without_side_effects(vm.names.parse).as_function();
```

Comment:
re build failure, I think this needs to be

```suggestion
    m_json_parse_function = &get_without_side_effects(vm.names.JSON).as_object().get_without_side_effects(vm.names.parse).as_function();
```

Reply:
Yeah just saw it, thanks. (Sorry for the very cooperative code review 😅)